pytest_tests/tests/container/test_container.py

```diff
@@ -211,3 +211,35 @@ def test_container_deletion_while_sn_down(
 
         with allure.step("Previously stopped node should return GC MARKED for the created object"):
             object_should_be_gc_marked(self.neofs_env, node_to_stop, cid, oid)
+
+    def test_container_global_name(self, default_wallet, simple_object_size):
```

Comment:
This test should be version-dependent, it fails with 0.43.0.

Reply:
fixed